### ISSUE TITLE
Compilation warning fixed, can you create a new release please?

### ISSUE THREAD

**Issue by jarvisjohnson** [kieraneglin/encrypted_secrets_ex#15]
Thanks for this package - great work!

We get this compilation warning about the `crypto` package:

```
warning: :crypto.strong_rand_bytes/1 defined in application :crypto is used by the current application but the current application does not depend on :crypto. To fix this, you must do one of:

  1. If :crypto is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :crypto is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :crypto, you may optionally skip this warning by adding [xref: [exclude: [:crypto]]] to your "def project" in mix.exs
```

I see it has been fixed in the master branch with `extra_applications: [:logger, :crypto]`

Could you create a new release version that includes this please?

thanks 🙏

**Comment by kieraneglin** [comment]
Good catch - thank you!  Version bumped and release published - 0.2.4 has what you need.

**Comment by jarvisjohnson** [comment]
Thanks @kieraneglin!
